migration/src/main/resources/META-INF/rewrite/axon-javax.yml

```diff
@@ -0,0 +1,55 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: org.axonframework.axon.UpgradeAxonFramework_4_7_Javax
+displayName: Upgrade to Axonframework 4.7 Javax
+description: This is what you may want if you don’t want Spring 6 and Spring Boot 3 support and expect things to stay the same.
+recipeList:
+  # Move all classes from org.axonframework.common.jpa to org.axonframework.common.legacyjpa
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: org.axonframework.common.jpa
+      newPackageName: org.axonframework.common.legacyjpa
+  # Move selected classes
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.jpa.DeadLetterJpaConverter
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.legacyjpa.DeadLetterJpaConverter
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.jpa.EventMessageDeadLetterJpaConverter
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.legacyjpa.EventMessageDeadLetterJpaConverter
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.jpa.JpaDeadLetter
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.legacyjpa.JpaDeadLetter
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.jpa.JpaSequencedDeadLetterQueue
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.legacyjpa.JpaSequencedDeadLetterQueue
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.tokenstore.jpa.JpaTokenStore
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.tokenstore.legacyjpa.JpaTokenStore
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.EmbeddedEventStore
```

Comment:
We can drop this class. We've made some adjustments in 4.7.1 that allow us to disregard this shift.

With that, I'll adjust the blog and RefGuide accordingly.

Reply:
Same as above: I've removed it, but if users are still on 4.7.0 is might make sense to keep this around.
```yml
  - org.openrewrite.java.ChangeType:
      oldFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.EmbeddedEventStore
      newFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.legacyjpa.EmbeddedEventStore
```

Reply:
Thanks again!